st2common/bin/bootstraprmq.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python2.7
+
+import sys
+import st2common.transport.bootstrap as transport_bootstrap
+
+if __name__ == '__main__':
```

Comment:
Not all consistency is a good idea :P

I really think that give our architecture each process ensuring the same indexes is actually a poor implementation choice. We really should have more control over some of these bootstrap steps.

But like I said I am making that change you ask for.